### PR TITLE
Fix php doc

### PR DESCRIPTION
**Pull request by Zeromax**
This PR proviedes lots of improvements for Popular IDEs like PHPStorm.
It also fixes some Imports which could break the code.

* fixed lots of php doc blocks
* Also some class imports

### CODE REVIEW

File: src/Query/Builder.php [hunk 1]
```diff
@@ -209,12 +212,14 @@ public function whereKey($id)
...
+     * @param string $first
+     * @param string $operator
+     * @param string $second
+     * @param string $type
+     * @param bool   $ref
+     * @param bool   $count
+     *
      * @return $this
      */
     public function expand($property, $first, $operator = null, $second = null, $type = 'inner', $ref = false, $count = false)
```
Comment (Zeromax) [inline src/Query/Builder.php]: I renamed `$property` in the docblock from `$table` to `$property`, But its never used.
But `$table` is used. Should I rename `$table` to `$property`?

Comment (anderly) [inline src/Query/Builder.php]: good catch. Yes, `$property` is more correct terminology for OData vs `$table` and I forgot to update the docblock. Let's go with `$property`.

Comment (Zeromax) [inline src/Query/Builder.php]: done in 97da34e

Comment (Zeromax) [inline src/Query/Builder.php]: There are some other errors in this method, I will create an other Issue for this

**Comment by Zeromax** [comment]
@anderly I'm done. Any other things which I should change?

**Comment by anderly** [comment]
@Zeromax can you squash commits and then I can merge?

**Comment by Zeromax** [comment]
@anderly opened a new PR for this #6 .
So you can close this, and merge the other one ;)

**Comment by anderly** [comment]
Closing in favor of #6.

**Pull request closed without merging**